routing_common/transit_types.cpp

```diff
@@ -0,0 +1,65 @@
+#include "routing_common/transit_types.hpp"
+
+#include "routing_common/transit_serdes.hpp"
+
+namespace routing
+{
+namespace transit
+{
+// TransitHeader ----------------------------------------------------------------------------------
+TransitHeader::TransitHeader(uint16_t version, uint32_t gatesOffset, uint32_t edgesOffset,
```

Comment:
It's up to you, but I don't think that this constructor is easier to use than, say,
```
TransitHeader h;
h.m_version = 0;
...
h.m_shapesOffset = 0xff;
...
h.CheckCompleteness();
```

Reply:
I guess I'd prefer to leave the constructor in this case. The thing is some default value of the header may be valid. So it's not an easy task to check if all fields are filled.